src/rezplugins/shell/gitbash.py

```diff
@@ -44,7 +44,7 @@ def find_executable(cls, name, check_syspaths=False):
                 "Git-bash executable has been detected at %s, but this is "
                 "probably not correct (google Windows Subsystem for Linux). "
                 "Consider adjusting your searchpath, or use rez config setting "
-                "plugins.shell.gitbash.executable_fullpath."
+                "plugins.shell.gitbash.executable_fullpath." % exepath
```

Comment:
Are you sure that this works? Should it be after the closing parenthesis?

Reply:
Yep it works, here's a sample output from my machine: 

```13:58:15 WARNING  Git-bash executable has been detected at c:\windows\system32\bash.exe, but this is probably not correct (google Windows Subsystem for Linux). Consider adjusting your searchpath, or use rez config setting plugins.shell.gitbash.executable_fullpath.```

Here's a couple other examples in the repo for reference:

https://github.com/AcademySoftwareFoundation/rez/blob/a13f7bb362cbb05e73f88e9f771c36005c1e9f4a/install.py#L83
https://github.com/AcademySoftwareFoundation/rez/blob/a13f7bb362cbb05e73f88e9f771c36005c1e9f4a/src/rez/cli/cp.py#L93
https://github.com/AcademySoftwareFoundation/rez/blob/a13f7bb362cbb05e73f88e9f771c36005c1e9f4a/src/rezgui/app.py#L19